client/state/stats/email-chart-tabs/selectors.js

```diff
@@ -50,10 +50,11 @@ export function isLoadingTabs(
 	barNumber
 ) {
 	const stats = get( state.stats.emails.items, [ siteId, postId, period, statType, date ], null );
-	// if we have redux stats ready return false
-	if ( dataLength < barNumber ) {
+	// If we have data for 30 columns, that's the maximum the endpoint can return
```

Comment:
Why did you make this change here? It seems like a valid assertion, but just wondering why :D

Reply:
Because the new width of the chart makes maxBars to 31, but that's bigger than the maximum 30 of the endpoint. Calling the endpoint 2 times for just 1 column out of 31 seems too much for me, so I made the email charts 30 columns maximum.